daemon/api_notices.go

```diff
@@ -213,3 +215,40 @@ func noticeViewableByUser(notice *state.Notice, requestUID uint32) bool {
 	}
 	return requestUID == userID
 }
+
+type noticeInfo struct {
```

Comment:
Why are having this structure again? Any reason we are not just adding json tags to public members in Notice?